docs/reference/quickwit-config.md

```diff
@@ -17,6 +17,7 @@ A commented example is accessible here: [quickwit.yaml](https://github.com/quick
 | --- | --- | --- |
 | version | Config file version. 0 is the only available value. |  |
 | node_id | Node ID of the instance (searcher or indexer). It must be unique in your cluster. If not set, a random ID is generated at each boot. |  |
+| cluster_name | Name for the cluster this node will be joining. Should be set to a unique name to ensure clusters do not accidentally merge together. | "Test cluster" |
```

Comment:
I wonder whether we should rename cluster_name to cluster_id to remain consistent with `index_id`, `node_id`, and `source_id`.

We probably also want to enforce the same naming convention (see #1007). This does not have to be tackled in this PR though.

Reply:
ah yes. can we make it `cluster_id` everywhere? .